gcp-gke-control-plane-auth-networks-updater/index.js

```diff
@@ -33,6 +33,8 @@ async function updateCluster(client, projectId, location, clusterName, mode, des
 
   logMasterAuthorizedNetworks(cluster.masterAuthorizedNetworksConfig.cidrBlocks)
 
+  waitForOperation(client, `projects/${ projectId }/locations/${ location }/operations/${ updateOp.name }`, 20)
```

Comment:
@ippschi-md Are you sure this works this way?

 I'm not very firm with js, but AFAICT the second parameter of the waitForOperation refers to a variable (`updateOp`) which is not yet defined, because it is defined in line 38 for the first time. 

I also wonder if the waitForOperation method actually works for operations that have been kicked of by a different instance of the code. From the [documentation](https://cloud.google.com/nodejs/docs/reference/container/5.9.0/container/v1.clustermanagerclient#_google_cloud_container_v1_ClusterManagerClient_getOperation_member_1_) it seems that it would return a promise, which I would expect to be a non-op if the running version of the code hasn't kicked of an async operation.

Reply:
@aptituz I am indeed confused, this is not working, this is only one code line. There is the part with the running operation missing